tests/src/tests/happy.rs

```diff
@@ -46,24 +46,28 @@ async fn happy_test(nodes: Vec<NomosNode>) {
     assert_eq!(blocks.len(), 1);
 }
 
-#[tokio::test]
-async fn two_nodes_happy() {
-    let nodes = NomosNode::spawn_nodes(SpawnConfig::Star {
-        n_participants: 2,
-        threshold: Fraction::one(),
-        timeout: Duration::from_secs(10),
-    })
-    .await;
-    happy_test(nodes).await;
+#[test]
+fn two_nodes_happy() {
+    tests::run_test(async {
+        let nodes = NomosNode::spawn_nodes(SpawnConfig::Star {
+            n_participants: 2,
+            threshold: Fraction::one(),
+            timeout: Duration::from_secs(10),
+        })
+        .await;
+        happy_test(nodes).await;
+    });
 }
 
-#[tokio::test]
-async fn ten_nodes_happy() {
-    let nodes = NomosNode::spawn_nodes(SpawnConfig::Star {
-        n_participants: 10,
-        threshold: Fraction::one(),
-        timeout: Duration::from_secs(10),
-    })
-    .await;
-    happy_test(nodes).await;
+#[test]
+fn ten_nodes_happy() {
```

Comment:
happy and unhappy already save logs upon failure, do you want to collect logs anyway?

Reply:
I would like to show logs in stdout, when debugging mutex blocking stuff yesterday, we only knew the test was timeout, but without any information in the CI log. So, I think having this will help debug in the future.

Reply:
WIth more than one process, it gets very confusing to show logs in stdout. We should instead add the possibility to download logs from CI